lite/examples/object_detection/android/app/src/androidTest/java/org/tensorflow/lite/examples/detection/DetectorTest.java

```diff
@@ -14,151 +14,150 @@
  * limitations under the License.
  */
 
-package org.tensorflow.lite.examples.detection;
-
```

Comment:
Please don't delete the test, but make it work.

Reply:
Check this example if it can help you:
https://android.googlesource.com/platform/frameworks/support/+/refs/heads/androidx-arch-core-release/camera/camera2/src/androidTest/java/androidx/camera/camera2/UseCaseCombinationTest.java

Reply:
Sure, will add this asap!

Reply:
@lintian06 @khanhlvg Test is not working in original project! It cannot loadRecognitions to compare with the output of the model. Test passes because no assertion is taking place. I have finished mocking media.Image in the new project and I am working on the post processing to compare the results.